api/api_net.go

```diff
@@ -73,5 +73,4 @@ type CommonNet interface {
 
 type NatInfo struct {
 	Reachability network.Reachability
-	PublicAddr   string
```

Comment:
Can we change this to an array, and populate this from Addrs? We have a bunch of tools / docs which want this API / CLI (mostly there some docs that refer to the Public address output from cli)